app/api/api_v1/routers/search.py

```diff
@@ -209,24 +210,23 @@ def download_all_search_documents(db=Depends(get_db)) -> RedirectResponse:
         # upload a buffer, it starts from the position it is currently in. We need to
         # add the seek(0) to reset the buffer position to the beginning before writing
         # to S3 to avoid creating an empty file.
-        df_as_csv = generate_data_dump_as_csv(INGEST_CYCLE_START, db)
-        df_as_csv.seek(0)
+        zip_buffer = create_data_download_zip_archive(INGEST_CYCLE_START, db)
+        zip_buffer.seek(0)
 
         try:
             response = s3_client.upload_fileobj(
-                bucket=DOC_CACHE_BUCKET,
-                key=data_dump_s3_key,
-                content_type="application/csv",
```

Comment:
I suspect we'll want to keep the content type, my understanding is it affects downloads knowing what kind of file it is. Be worth double checking